libs/openFrameworks/math/ofQuaternion.h

```diff
@@ -199,21 +199,21 @@ ofQuaternion::ofQuaternion() {
 
 
 //----------------------------------------
-ostream& ofQuaternion::operator<<(ostream& os) {
-    os << _v.x << ", " << _v.y << ", " << _v.z << ", " << _v.w;
+ostream& operator<<(ostream& os, const ofQuaternion &q) {
```

Comment:
or add inline to the implementation but for something like this probably better to just move it to the cpp